pcap-win32.c

```diff
@@ -1061,9 +1087,22 @@ pcap_activate_win32(pcap_t *p)
 	return (0);
 bad:
 	pcap_cleanup_win32(p);
+	if (p->opt.rfmon_selfstart)
```

Comment:
It has been fixed at: https://github.com/the-tcpdump-group/libpcap/pull/512/commits/1c3bf75b09e74efaf63179ef8aba000d2610b415